lib/topic_publisher.rb

```diff
@@ -46,6 +46,14 @@ def publish!
       end
     end
 
+    Jobs.enqueue(
+      :notify_tag_change,
+      post_id: @topic.first_post.id,
+      notified_user_ids: [@topic.first_post.user_id],
```

Comment:
Should `@published_by.id` be included in `notified_user_ids` or is this handled somewhere else? I think it could be weird if I'm watching a tag and get notified when I publish a topic with that tag.

Reply:
That makes a lot of sense. I made the changes.